src/main/java/step2/AutoLottoController.java

```diff
@@ -0,0 +1,85 @@
+package step2;
+
+import step2.domain.AutoLottoMatchCount;
+import step2.domain.Lotto;
+import step2.domain.Prize;
+import step2.domain.User;
+import step2.util.PrizeCalculatorUtil;
+import step2.util.StringUtil;
+import step2.view.InputView;
+import step2.view.ResultView;
+
+import java.util.HashMap;
+import java.util.Iterator;
+
+public class AutoLottoController {
+
+    InputView inputView;
+    ResultView resultView;
+    User user;
+    Lotto defaultLotto;
+    AutoLottoMatchCount autoLottoMatchCount;
```

Comment:
전체적으로 접근제어자가 필요이상으로 넓어요. 접근제어자 키워드를 사용해서 최대한 캡슐화해보세요.
final 키워드를 사용하여서 불변임을 보장하는 것도 좋은 습관이라고 생각해요. 🙂 
```suggestion
    private final InputView inputView;
    private final ResultView resultView;
    private final User user;
    private final Lotto defaultLotto;
    private final AutoLottoMatchCount autoLottoMatchCount;
```